pkg/vadmin/restore_points_query_at.go

```diff
@@ -0,0 +1,25 @@
+/*
+ (c) Copyright [2021-2023] Open Text.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package vadmin
+
+import (
+	"context"
+)
+
+func (a *Admintools) ListRestorePoints(_ context.Context) error {
+	a.Log.Info("start restore points query")
```

Comment:
I don't think we will ever implement this in the admintools side. Lets return an error if this is ever called.